### PR TITLE
o2-eve-display track<-> cluster relation

### PR DESCRIPTION
**Pull request by jmyrcha**
changes in format of *.json file
o2-eve will display old/new format for a while

### CODE REVIEW

**Comment by jmyrcha** [comment]
build/O2/o2-dataflow-cs8 failed on unrelated code

**Comment by jmyrcha** [comment]
build/O2/o2-cs8 also failed on unrelated code

File: EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h [hunk 1]
```diff
@@ -81,11 +81,10 @@ class VisualisationEvent
     return mClusters.back();
   }
 
-  VisualisationCluster& addCluster(float X, float Y, float Z, float time)
+  VisualisationCluster& addCluster(float X, float Y, float Z, float trackTime)
```
Comment (shahor02) [inline EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h:83]: You have 2 methods `addCluster` with almost similar signature doing completely different things, which is error prone. Perhaps this one should be `addTrackCluster (..)`?

Comment (jmyrcha) [reply EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h:83]: This is error - I will fix that

**Comment by pzhristov** [comment]
The errors are unrelated, merging.

**Pull request merged**